pocs/core.py

```diff
@@ -312,8 +312,12 @@ def is_safe(self, no_warning=False, horizon='observe', **kwargs):
         # Check weather
         is_safe_values['good_weather'] = self.is_weather_safe()
 
-        # Hard-drive space
-        is_safe_values['free_space'] = self.has_free_space()
+        # Hard-drive space in root
+        is_safe_values['free_space_root'] = self.has_free_space()
```

Comment:
```suggestion
        is_safe_values['free_space_root'] = self.has_free_space('/')
```

Let's explicitly check for `/`